resources/gnomad_liftover.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+from basics import *
+from gnomad_hail import *
+import logging
+
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("gnomAD_liftover")
+logger.setLevel(logging.INFO)
+
+
+def checkpoint(ht: hl.Table, dtype: str) -> hl.Table:
```

Comment:
This function now exists in hail! `ht = ht.checkpoint(path)` does exactly this

Reply:
that's awesome, I had no idea! will remove this function

Reply:
It's brand new (as of 0.2.11 I think). I also added a hidden option not to rerun stuff above if the file already exists `ht = ht.checkpoint(path, _read_if_exists=True)` (note that if you change the code it won't regenerate the file, but it's good for fast iteration on downstream things)